GOVERNANCE.md

```diff
@@ -0,0 +1,62 @@
+# Benevolent Dictatorship Governance
+This document describes the governance model for this project.
+
+## Overview
+This project is led by a benevolent dictator and managed by the community. That is, the community actively contributes to the day-to-day maintenance of the project, but the general strategic line is drawn by the benevolent dictator. In case of disagreement, the benevolent dictator has the the last word. It is the benevolent dictator’s job to resolve disputes within the community and to ensure that the project is able to progress in a coordinated way. In turn, it is the community’s job to guide the decisions of the benevolent dictator through active engagement and contribution.
```

Comment:
nit: double `the`
`has the the last word`

Reply:
done;